virt/virt-2-5-release-notes.adoc

```diff
@@ -64,6 +64,8 @@ Other operating system templates shipped with {VirtProductName} are not supporte
 //Placeholder for new content.
 
 //CNV-7170 - CDI import of container disks is faster and more efficient
+You can now use the Containerized Data Importer (CDI) to faster import `containerDisk` storage volumes from the container image registry and allocate storage capacity more efficiently. The CDI can pull a `containerDisk` image from the registry in about the same amount of time that it would
+take for an HTTP import. The imported image is embedded in a PersistentVolumeClaim (PVC) equal in size to the disk image to use CDI scratch space and the underlying storage more efficiently.
```

Comment:
I don't think it's necessary to mention scratch space in the release note.  Instead I would say something like the following:

When importing, you may use a PersistentVolumeClain (PVC) size equal to the disk image size.